ui/src/PrivateLayout.scss

```diff
@@ -1,3 +1,7 @@
 .main-component-layout {
   padding-top: 49px;
 }
+
+.css-1asjec2-euiPageSection__content-l-restrictWidth {
```

Comment:
Is it something temporary? You are not intended to modify the `@emotion/react` CSS styles directly. Also, I'm not sure if this CSS class name is stable (you can guess it from this generated name `.css-1asjec2-...`)

Reply:
Yeah it is something temporary, I was just publishing the image temporarily to test out something else; but I'm going to fix this before opening this PR.